bundles/org.openhab.binding.nikohomecontrol/src/main/resources/OH-INF/i18n/nikohomecontrol_nl.properties

```diff
@@ -24,8 +24,8 @@ bridge2ConfigPasswordLabel = API Token
 bridge2ConfigPasswordDescription = Token voor Niko Home Control II hobby API, mag niet leeg zijn. Dit token moet worden verlengd na vervaldatum (1 jaar na aanmaak)
 
 bridgeConfigRefreshLabel = Verversingsinterval
-bridgeConfigRefreshDescription = Verversingsinterval voor verbinding met Niko Home Control IP-interface (min), standaard 300. Indien 0 is ingesteld of leeg gelaten, zal er geen verversing worden gepland
-bridge2ConfigRefreshDescription = Verversingsinterval voor verbinding met Connected Controller of Draadloze Slimme Hub (min), standaard 300. Als 0 of leeg is, wordt er geen verversing gepland
+bridgeConfigRefreshDescription = Verversingsinterval voor verbinding met Niko Home Control IP-interface (min), standaard 300. Indien 0 is ingesteld  zal er geen verversing worden gepland
```

Comment:
Not to be corrected in that PR but rather later in Crowdin.

Reply:
OK